### PR TITLE
The WX rework! Changes lots of stuff from the way charge system works to circuits and their effects :]

### PR DESCRIPTION
**Pull request by FourthLess**
Basically stuff from WX:Re-wired except for the circuit box since it's not finished yet :]
Config for it and stuff is also there
Atober please accept 👍 💯 👍 💯

### CODE REVIEW

**Comment by FourthLess** [comment]
If it breaks uhhh
don't blame the player, blame the game

File: modinfo.lua [hunk 1]
```diff
@@ -231,6 +231,8 @@ configuration_options = {
         --"Willow's insulation is tweaked to be 120 on Summer and -120 on Winter.", false),
     BinaryConfig("wendy", "Wendy", "Abigail is nerfed to not increase Wendy's maximum damage above average.", true),
     BinaryConfig("wx78", "WX-78", "No longer heals from lightning.", true),
+    BinaryConfig("wxless", "WX Rework",
+        "Changes the way WX's charge and circuit systems works, alongside the effects of circuits", false),
```
Comment (AtobaAzul) [inline modinfo.lua:235]: It could probably be left on-by-default.

File: init/init_character_changes/wx78.lua [hunk 2]
```diff
@@ -1,53 +1,644 @@
...
+
+    if TUNING.DSTU.WX78_CONFIG then
+		if inst.components.playerlightningtarget ~= nil then
+			inst.components.playerlightningtarget:SetOnStrikeFn(OnLightningStrike)
+		end
+    end
+end)
+
+-------------WX RE-WIRED CHANGES :]------------------
+if TUNING.DSTU.WXLESS then --HI ATOBA :3 :3 <3 <3
```
Comment (AtobaAzul) [inline init/init_character_changes/wx78.lua:56]: Hi!!

**Review by AtobaAzul — APPROVED** [review]
Alrighty, sounds good! Could probably be on-by-default though!

**Pull request merged**